SWLOR.Game.Server/Perk/ForceAlter/ForceStun.cs

```diff
@@ -146,42 +146,45 @@ private void RunEffect(NWCreature creature, NWObject target)
 
         private void ApplyEffect(NWCreature creature, NWObject target, int spellTier)
         {
-            const float radiusSize = 10.0f;
+            const float radiusSize = 6.0f;
             NWCreature targetCreature;
             
             switch (spellTier)
             {
                 // Tier 1 - Single target is Tranquilized or, if resisted, receives -5 to AB and AC
                 case 1:
-                    RunEffect(creature, target);
+                        RunEffect(creature, target);
                     break;
                 // Tier 2 - Target and nearest other enemy within 10m are tranquilized using tier 1 rules.
                 case 2:
                     RunEffect(creature, target);
                     
                     // Target the next nearest creature and do the same thing.
-                    targetCreature = GetFirstObjectInShape(Shape.Sphere, radiusSize, creature.Location, true);
+                    targetCreature = GetFirstObjectInShape(Shape.Sphere, radiusSize, target.Location, true);
                     while (targetCreature.IsValid)
                     {
                         if (targetCreature != target)
                         {
                             // Apply to nearest other creature, then exit loop.
-                            RunEffect(creature, target);
+                            if (_.GetIsReactionTypeHostile(creature, targetCreature)) 
+                                RunEffect(creature, targetCreature);
+                            else
```

Comment:
If you don't find a hostile target, you're exiting this while condition. That doesn't seem right to me. Is it? If it is, tab over the break; statement. If it isn't, remove the else.